crates/rpc/rpc-api/src/debug.rs

```diff
@@ -90,6 +90,17 @@ pub trait DebugApi {
         opts: Option<GethDebugTracingOptions>,
     ) -> RpcResult<GethTrace>;
 
+    /// The `debug_traceTransaction` debugging method will attempt to run the transaction in the
+    /// exact same manner as it was executed on the network. It will replay any transaction that
+    /// may have been executed prior to this one before it will finally attempt to execute the
+    /// transaction that corresponds to the given hash.
+    #[method(name = "traceTransactionOverrides")]
+    async fn debug_trace_transaction_overrides(
+        &self,
+        tx_hash: B256,
+        opts: Option<GethDebugTracingCallOptions>,
```

Comment:
I think this is the same as adding support for overrides in debug_trace_transaction?

`GethDebugTracingCallOptions` contains two optional objects but is otherwise the same as `GethDebugTracingOptions`

so I think we can integrate this without breaking anything and avoid a new endpoint

Reply:
it is, but as I mentioned this distinction is to avoid confusion.
also see this reply in my geth version:
https://github.com/ethereum/go-ethereum/pull/29423#issuecomment-2029641832

it seems they support this within debug_traceCall (less convenient imo) but I've never seen this documented for any node before.

I want a standardized api across nodes, each node makes their own modifications and then things break when, for example, using load balancers across different providers.